node/silkworm/downloader/packets/NewBlockHashesPacket.hpp

```diff
@@ -21,18 +21,17 @@
 
 namespace silkworm {
 
-    struct NewBlockHash {     // one particular block being announced
-        Hash hash;        // hash of the block
-        BlockNum number;  // number of the block
-    };
+struct NewBlockHash {  // one particular block being announced
+    Hash hash;         // hash of the block
+    BlockNum number;   // number of the block
+};
 
-    using NewBlockHashesPacket = std::vector<NewBlockHash>;
+using NewBlockHashesPacket = std::vector<NewBlockHash>;
 
 namespace rlp {
 
     inline void encode(Bytes& to, const NewBlockHash& from) noexcept {
-        rlp::Header rlp_head{true,
-                             rlp::length(from.hash) + rlp::length(from.number)};
+        rlp::Header rlp_head{true, rlp::length(ByteView{from.hash}) + rlp::length(from.number)};
```

Comment:
The formatting changes had prevented me from seeing the semantic change from `rlp::length(Hash)` to 
`rlp::length(ByteView)` (perhaps it would be better for the reviewer to have different changes in different commits).
Anyway why this change? 

We should carefully consider whether to add this overload
`constexpr ByteView(const evmc::bytes32& hash) noexcept;`
I feel it causes overload resolution problems elsewhere